cmd/syncProcessor.go

```diff
@@ -106,6 +130,26 @@ func (d *interactiveDeleteProcessor) removeImmediately(object storedObject) (err
 		return nil
 	}
 
+	if d.dryrunMode {
+		glcm.Dryrun(func(format common.OutputFormat) string {
+			if format == common.EOutputFormat.Json() {
+				jsonOutput, err := json.Marshal(newDeleteTransfer(object))
+				common.PanicIfErr(err)
```

Comment:
Same as [this](https://github.com/Azure/azure-storage-azcopy/pull/1495/files#r672995443)